### ISSUE TITLE
Add country to info call

### ISSUE THREAD

**Issue by lil5** [the-clothing-loop/website#403]
This requires loops to have a location country code field

- [x] Depends on #394 
- [x] Add `country_code` column to `chains` table in database
- [x] Set country code, either by long lat (go) or via mapbox (js)
- [x] Add sql to count unique values in the `country_code` column, to the info call
- [x] Set the result in the website

**Comment by tobifaf** [comment]
Hi @lil5, I'd like to start contributing to this project. Is this issue up for grabs?

**Comment by lil5** [comment]
@tobifaf Definitely go a head 🚀

**Comment by tobifaf** [comment]
I was leaning towards using mapbox to get the country code, since it's already returned in "getPlaceName", but I suspect you want to also want to add the country code to existing loops? Did you add a field like this before?

**Comment by lil5** [comment]
@tobifaf 

> I was leaning towards using mapbox to get the country code, since it's already returned in "getPlaceName", 
Yes, I agree, using existing values from mapbox in the frontend is probably best.
Then sending the country code to the backend.

> but I suspect you want to also want to add the country code to existing loops?
This will be fixed by a separate issue, I might just write some SQL for that

**Comment by tobifaf** [comment]
I created a PR with a proposal. I couldn't get the test suite to work, do you have any documentation on that?

**Comment by lil5** [comment]
The GoLang tests require a copy of `config.dev.yml` to `config.test.yml`. Then you should be able to run tests

**Comment by tobifaf** [comment]
> The GoLang tests require a copy of `config.dev.yml` to `config.test.yml`. Then you should be able to run tests

Great, I got it working

**Comment by lil5** [comment]
@tobifaf nice one! If you want you can join the dev meeting I host, if your interested email me at li@last.nl
